src/main/java/htsjdk/variant/variantcontext/writer/VariantContextWriterBuilder.java

```diff
@@ -484,11 +484,6 @@ else if (STREAM_TYPES.contains(this.outType))
                 writer = createBCFWriter(outPath, outStreamFromFile);
                 break;
             case VCF_STREAM:
-                if (options.contains(Options.INDEX_ON_THE_FLY)) {
```

Comment:
why does this have to be removed?

Reply:
So a tabix index can be written when writing a VCF to a stream.